pkg/e2e/verify/pod_webhook.go

```diff
@@ -250,3 +269,35 @@ func impersonateDedicatedAdmin(h *helper.H, user string) *helper.H {
 
 	return h
 }
+
+func getDaemonSet(namespace string, daemonSetName string, h *helper.H) wait.ConditionFunc {
+	return func() (bool, error) {
+		_, err := h.Kube().AppsV1().DaemonSets(namespace).Get(context.TODO(), daemonSetName, metav1.GetOptions{})
+		Expect(err).NotTo(HaveOccurred(), "failed to get daemonset %v\n", daemonSetName)
+		if err != nil {
+			return false, err
+		}
+		return true, nil
+	}
+}
+
+// waitTimeoutForDaemonSetInNamespace waits the given timeout duration for the specified ds.
+func waitTimeoutForDaemonSetInNamespace(h *helper.H, daemonSetName string, namespace string, timeout time.Duration) error {
```

Comment:
Could I convince you to relocate this to a new method on the Helper type? `func (h *Helper) WaitForDaemonSet()` (and do the same for the service polling function below)? They seem useful to many people writing tests, not just this specific use-case.

Reply:
@whereswaldon thanks for the suggestion!  I updated the commit accordingly.